styles.css

```diff
@@ -458,3 +460,399 @@ form {
 .message-label {
   padding-bottom: 40%;
 }
+
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media only screen and (max-width: 1440px) and (min-width: 768px) {
+  /* headliner page */
+  .toolbar {
+    padding-left: 10%;
+    width: 90vw;
+    margin-right: 0;
+  }
+
+  .toolbar-links {
+    color: #344563;
+    text-decoration: none;
+    display: flex;
+    justify-content: start;
+    width: 20vw;
+    gap: 4%;
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .toolbar-links > a {
+    padding: 0;
+    margin: 0;
+  }
+
+  #work {
+    height: 320vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+  }
+
+  .wrapper-1 {
+    height: 100vh;
+    background-color: #f7f7f9;
+  }
+
+  .headline {
+    background-image: url(images/desktop-pictures/Header\ bg.png), url(images/desktop-pictures/header\ background\ shape-desktop.svg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: 105.9%;
+    padding: 20%;
+    padding-top: 0;
+    padding-bottom: 0;
+    border-radius: 0 0 0 180px;
+    height: 68vh;
+  }
+
+  .headline > * {
+    padding: 12px;
+  }
+
+  .wrapper-container {
+    margin: 0;
+    padding: 0;
+  }
+
+  /* work section */
+  .project {
+    padding: 10%;
+    padding-bottom: 0;
+    height: 65vh;
+  }
+
+  .container {
+    border: 1px solid #dfe1e6;
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: repeat(6, 1fr);
+    grid-gap: 0;
+    height: 53vh;
+    padding: 2rem;
+  }
+
+  .project-1 {
+    order: 3;
+  }
+
+  .project-2 {
+    order: 4;
+    padding-bottom: 10%;
+  }
+
+  .project-3 {
+    order: 1;
+  }
+
+  .project-3 > .container {
+    width: 75vw;
+  }
+
+  .project-4 {
+    order: 2;
+  }
+
+  .container > * {
+    margin: 0;
+    padding-left: 2vw;
+  }
+
+  .container > img {
+    width: 100%;
+    grid-row: 1 / 7;
+    padding: 3%;
+    padding-right: 0;
+  }
+
+  .image-desktop,
+  .li-desktop,
+  .project-desktop,
+  .client-name-desktop,
+  .name-desktop {
+    display: block;
+  }
+
+  .li-mobile,
+  .project-mobile,
+  .client-name-mobile,
+  .image-mobile {
+    display: none;
+  }
+
+  .container > .flip-img {
+    grid-column: 2 / 3;
+  }
+
+  .client-name {
+    height: 50%;
+    align-self: end;
+    margin: 2%;
+    font-size: 2vw;
+  }
```

Comment:
- [ ] Ouch! this is close as well but it seems like the headings in your project card have a smaller font size as compared to the Figma design as you can see the font size is specified to be `40px` so kindly adjust it so it looks similar to the design.
- [ ] Your design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/e78b1540-1042-40af-87d7-39e8211291d8)
- [ ] Figma design:
    ![image](https://github.com/agneta1991/Portfolio/assets/104968236/d504ddb1-7022-42dc-b125-4078dfee6e11)